README.md

```diff
@@ -92,6 +104,10 @@ spec:
 * If needed, you can use [ConfigMap](http://kubernetes.io/docs/user-guide/configmap/)
 to overwrite the `config/`.
 
+## Start Standalone
+`inClusterConfig` should be set to `false`. To run node-problem-detector standalone with an insecure apiserver connection: 
+`node-problem-detector -apiserver-override=http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false`
```

Comment:
To run node-problem-detector standalone, you should set `inClusterConfig` to `false` and teach node-problem-detector how to access apiserver with `apiserver-override`.

To run node-problem-detector standalone with an insecure apiserver connection:
```
node-problem-detector -apiserver-override=http://APISERVER_IP:APISERVER_PORT?inClusterConfig=false
```

For more scenarios, see [here](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md#kubernetes)